cpu/cc2538/periph/spi.c

```diff
@@ -132,6 +131,7 @@ static uint8_t _trx(cc2538_ssi_t *dev, uint8_t in)
 void spi_transfer_bytes(spi_t bus, spi_cs_t cs, bool cont,
                         const void *out, void *in, size_t len)
 {
+    assert((unsigned)bus < SPI_NUMOF);
```

Comment:
Why assert in spi_transfer_bytes instead of spi_acquire for this one ?

Reply:
See https://github.com/RIOT-OS/RIOT/pull/16815